app/src/main/kotlin/cz/covid19cz/erouska/exposurenotifications/ExposureNotificationsRepository.kt

```diff
@@ -0,0 +1,80 @@
+package cz.covid19cz.erouska.exposurenotifications
+
+import android.bluetooth.BluetoothAdapter
+import com.google.android.gms.nearby.exposurenotification.ExposureNotificationClient
+import com.google.android.gms.nearby.exposurenotification.ExposureWindow
+import com.google.android.gms.nearby.exposurenotification.TemporaryExposureKey
+import cz.covid19cz.erouska.db.SharedPrefsRepository
+import java.io.File
+import kotlin.coroutines.resume
+import kotlin.coroutines.resumeWithException
+import kotlin.coroutines.suspendCoroutine
+
+class ExposureNotificationsRepository(
+    private val exposureNotificationClient: ExposureNotificationClient,
+    private val btAdapter: BluetoothAdapter,
+    private val prefs: SharedPrefsRepository
+) {
+
+    fun isBluetoothEnabled(): Boolean {
+        return btAdapter.isEnabled
+    }
+
+    suspend fun start() = suspendCoroutine<Void> { cont ->
```

Comment:
Ten <Void> tady je potřeba?

Reply:
jj, tím se zapínají exposure notifications

Reply:
V komentáře není vidět `<Void>`, na to jsem se ptal

Reply:
jj, protože exposureNotificationClient.start() vrací Task<Void>